src/store/index.ts

```diff
@@ -14,11 +14,15 @@ import mutations from './mutations';
 import RootState from './RootState';
 
 interface StoreParams {
-	token?: string;
+	token?: string,
+	licenseUrl?: string,
+	licenseName?: string;
```

Comment:
We have two lines ending with `,` and one ending with `;` here now, that seems inconsistent to me. (Would be nice to have an eslint rule for it, to fix it automatically.)

Reply:
Now in #77.